### PR TITLE
add emulation less file

### PR DESCRIPTION
**Pull request by ajnassar**

### CODE REVIEW

**Comment by ajnassar** [comment]
@jontsai Here is the PR for emulate_user.less

File: emulate_user.less [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.htk-toolbar {
```
Comment (jontsai) [inline emulate_user.less:1]: can every rule in here be nested inside of some parent/container class?

**Review by jontsai — CHANGES_REQUESTED** [review]

File: emulate_user.less [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    overflow-y: auto;
+    right: 0;
+    top: 0;
+    width: 200px;
+    z-index: 100000000;
+
+    opacity: 0.9;
+    background-color: #111;
+    .htk-toolbar-hide {
+        border-bottom: 1px solid #222;
```
Comment (jontsai) [inline emulate_user.less:14]: all of the hardcoded color codes here should be replaced with variables later

**Review by jontsai — APPROVED** [review]
LFTM

File: emulate_user.less (hunk 2 above)
Comment (ajnassar) [reply emulate_user.less:14]: Roger that

**Pull request merged**